tests/vmi_configuration_test.go

```diff
@@ -561,11 +561,13 @@ var _ = Describe("Configurations", func() {
 				Expect(err).ToNot(HaveOccurred())
 				defer expecter.Close()
 
-				res, err := expecter.ExpectBatch([]expect.Batcher{
+				res, err := tests.ExpectBatchWithValidatedSend(expecter, []expect.Batcher{
 					&expect.BSnd{S: "[ $(free -m | grep Mem: | tr -s ' ' | cut -d' ' -f2) -gt 200 ] && echo 'pa''ss'\n"},
-					&expect.BExp{R: "pass"},
-					&expect.BSnd{S: "swapoff -a && dd if=/dev/zero of=/dev/shm/test bs=1k count=118k; echo $?\n"},
-					&expect.BExp{R: "0"},
+					&expect.BExp{R: tests.Retcode("pass", "\\$ ")},
```

Comment:
maybe worth to rename Retcode so it will fit to strings and retcodes ?

Reply:
What name do you suggest? RetVal?

Reply:
RetString ?
RetValue

Reply:
Changed to RetValue